ldap/servers/slapd/dse.c

```diff
@@ -2055,12 +2057,28 @@ dse_modify(Slapi_PBlock *pb) /* JCM There should only be one exit point from thi
                 }
             }
 
+            plugin_call_plugins(pb, SLAPI_PLUGIN_BE_POST_MODIFY_FN);
+            if (!returncode) {
+                slapi_pblock_get(pb, SLAPI_RESULT_CODE, &returncode);
+            }
+        }
+    } else {
+        /* It should not happen but just be paranoiac, do not
+         * forget to call the postop if needed
+         */
+        if (need_be_postop) {
```

Comment:
Is there a situation where this is false?

Reply:
Yes there are several "early" 'goto done;' in the function where the dse_modify fails before preop are called. For example if the modified entry does not exist (LDAP_NO_SUCH_OBJECT)